server/routes/api/auth/auth.test.ts

```diff
@@ -93,10 +93,11 @@ describe("#auth.config", () => {
     const res = await server.post("/api/auth.config");
     const body = await res.json();
     expect(res.status).toEqual(200);
-    expect(body.data.providers.length).toBe(3);
+    expect(body.data.providers.length).toBe(4);
     expect(body.data.providers[0].name).toBe("Slack");
     expect(body.data.providers[1].name).toBe("OpenID Connect");
     expect(body.data.providers[2].name).toBe("Google");
+    expect(body.data.providers[3].name).toBe("Github");
```

Comment:
There's a subtle distinction between GitHub and others - others are also used for user login but GitHub is expected to be used only as an integration.

Reply:
For this reason let's keep /auth routes for login options only. It's ok in the main api namespace

Reply:
Alternatively can register the route at the /auth/ path but don't out the plugin logic in the auth folder. 

Just don't want it picked up as a login provider

Reply:
Oops! I think we ought to exclude `github` here as we've done for `email`...

Reply:
> Alternatively can register the route at the /auth/ path but move the plugin logic out of the auth subdirectory.

Uff, this would require modifying the dynamic import logic. Moving under `/api` is easier 😅